src/jdk.jdeps/share/classes/com/sun/tools/jdeps/JdepsTask.java

```diff
@@ -790,8 +790,8 @@ boolean run(JdepsConfiguration config, JdepsWriter writer, Type type)
                     String replacementApiTitle = getMessage("public.api.replacement.column.header");
                     log.format("%-40s %s%n", internalApiTitle, replacementApiTitle);
                     log.format("%-40s %s%n",
-                               internalApiTitle.replaceAll(".", "-"),
-                               replacementApiTitle.replaceAll(".", "-"));
+                               "-".repeat(internalApiTitle.length()),
+                               "_".repeat(replacementApiTitle.length()));
```

Comment:
Using replaceAll to replace all characters with "-" is indeed a bit unusual. Are you sure String::length is what you want, I'm just wondering if you need the number of Unicode code points  here rather than length.

Reply:
These seem to be strings from localization, so I think it's unlikely to have 4-byte unicode characters appearing.